test/integrations/destinations/iterable/router/data.ts

```diff
@@ -852,6 +867,7 @@ export const data = [
                 },
                 Config: {
                   apiKey: '12345',
+                  dataCenter: 'USDC',
```

Comment:
The tests should work without this as well right ? Do we really need to add this ?

Reply:
Add this in one(successful scenario) & remove for other test-cases.

Reply:
@sanpj2292 dataCenter is a required field. And also using dataCenter only endpoint will be constructed. Therefore we'll need to keep it, otherwise test cases will fail.

Reply:
Currently I don't think Iterable destination has a field for data-center. Are we planning any migration to insert dataCenter field inside destination configuration for existing iterable destinations ?

Reply:
Yes, I've added dataCenter as a field in the existing iterable destination configuration only. User can choose between USDC or EUDC. 
https://github.com/rudderlabs/rudder-integrations-config/pull/1762